### PR TITLE
Use latest released SDK.

### PR DESCRIPTION
**Pull request by xingwu1**

### CODE REVIEW

File: Java/PoolAndResourceFile/pom.xml [hunk 1]
```diff
@@ -11,37 +11,13 @@
 
   <groupId>com.microsoft.azure</groupId>
   <artifactId>PoolAndResourceFile</artifactId>
-  <version>1.0-SNAPSHOT</version>
```
Comment (darylmsft) [inline Java/PoolAndResourceFile/pom.xml]: hmm what is a snapshot?  I guess they are no longer neeeded now that this is transitioning to ga?

File: Java/PoolAndResourceFile/pom.xml [hunk 2]
```diff
@@ -11,37 +11,13 @@
...
-    </repository>
-  </repositories> 
+  <version>1.0.0</version>
 
   <dependencies>
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-batch</artifactId>
-      <version>1.0.0-rc</version>
+      <version>1.0.0-rc2</version>
```
Comment (darylmsft) [inline Java/PoolAndResourceFile/pom.xml]: oh -rc2... we are not ga?

Comment (xingwu1) [inline Java/PoolAndResourceFile/pom.xml]: No, I found more runtime/autorest bugs (new rc2 SDK include the workaround). Wait the fix in at end of March.

File: Java/PoolAndResourceFile/pom.xml (hunk 1 above)
Comment (xingwu1) [inline Java/PoolAndResourceFile/pom.xml]: This is sample's version.

**Pull request merged**